whisper/whisperv6/whisper.go

```diff
@@ -176,14 +181,41 @@ func (w *Whisper) SetMaxMessageSize(size uint32) error {
 }
 
 // SetMinimumPoW sets the minimal PoW required by this node
-func (w *Whisper) SetMinimumPoW(val float64) error {
+func (w *Whisper) SetMinimumPoW(val float64, testMode bool) error {
 	if val <= 0.0 {
 		return fmt.Errorf("invalid PoW: %f", val)
 	}
-	w.settings.Store(minPowIdx, val)
+
+	w.notifyPeersAboutPowRequirementChange(val)
+
+	if testMode {
```

Comment:
Why making such a big difference between the tests and the regular code? If the real function waits, the tests should too.

Reply:
real function waits for a reason, which is realated to syncronization in real-life environment. but if this test will run longer than all other tests together, than it will be really painful to run the tests every time.

Reply:
I understand it might take longer to run the test, at the same time running the whole list of tests is really long because of swarm so I would not lose sleep over this. What you can also do, is configure the waiting time to be different in the case of a test environment, so the code remains the same but the wait time is shorter for tests. That would ensure that the code that is being tested is the same as that which runs in production.

I have been looking for a way to distinguish a normal build from a test build in the doc, haven't found anything so far. Anything else that the powers that be have decided you don't need?

If you manage to do that, then my recommendation in the previous comment doesn't need to happen.